tests/test_aws.py

```diff
@@ -365,10 +365,20 @@ def test_copy_ami(self):
         self.mock_copy_ami.return_value = {
             'ImageId': 'ami-08',
             'ResponseMetadata': {'RequestId': 'cf32',  'server': 'AmazonEC2'}}
-        copy_ami_result = self.svc.copy_ami("ami-01", "rhcos", "us-east-1")
-        self.assertEqual(copy_ami_result['ImageId'], "ami-08")
+        copy_ami_result = self.svc.copy_ami(
```

Comment:
Minor: rather than changing the only test which calls `copy_ami` so that it passes the new argument, it would be nice to instead add a new call so that there is coverage of both passing and not passing `tags`.

This relates to my earlier comment about the argument wrongly not being optional.  That'll cause existing callers of `copy_ami` to crash.  The test not passing `tags` accurately reproduces what any existing caller of `copy_ami` was doing, so the test would have been able to catch this crash.

Changing the test so it always passes `tags` means there is no longer any test which reproduces what the current users of `copy_ami` are doing, hiding a potential source of crash that real users would encounter.

Reply:
Changes it to support cases with tags as None.

Reply:
@rohanpm PTAL